content/300-guides/050-application-lifecycle/100-developing-with-prisma-migrate/100-advanced-migrate-scenarios.mdx

```diff
@@ -0,0 +1,195 @@
+---
+title: Advanced migration scenarios
+---
+
+<TopBlock>
+
+* Create a draft
+* Modify SQL
+* `--create-only` flag
+
+</TopBlock>
+
+## Renaming a field
```

Comment:
A similar approach can be used for renaming models, instead of dropping, re-creating the table, just alter the name.

Reply:
Maybe we could expand this to cover both or we could add a separate section for that, now or later.

Reply:
I'll add a note for now and then we can cover later!